src/main/java/com/lunark/lunark/model/ReviewReport.java

```diff
@@ -0,0 +1,37 @@
+package com.lunark.lunark.model;
+
+import java.time.LocalDateTime;
+
+public class ReviewReport {
+    private Long id;
+    private LocalDateTime date;
+
```

Comment:
Shouldn't ReviewReport reference Review?
```suggestion
private Review review;
```

Reply:
You're right. I will refactor the code soon.